doc/cli.markdown

```diff
@@ -982,10 +982,6 @@ Examples:
 
 ### Options
 
-#### --all
-
-No-op since balena CLI v12.0.0.
-
```

Comment:
This should not have been in the docs, since it was a hidden field (capitanodoc does not currently respect the `hidden` property for flags)